docs/api-reference/map-controller.md

```diff
@@ -15,6 +15,17 @@ onViewportChange={v => this.setState({viewport: v})}
 />
 ```
 
+## Options
```

Comment:
Should we mention that the options are not used here, they are used on `View`? And example above is probably wrong?

Reply:
The example's use case is still supported (specifying controller without creating a view).